crates/viewer/re_space_view/src/annotation_context_utils.rs

```diff
@@ -0,0 +1,137 @@
+use ahash::HashMap;
+
+use re_types::components::Color;
+use re_viewer_context::{Annotations, QueryContext, ResolvedAnnotationInfos};
+
+use crate::clamped_or_nothing;
+
+#[inline]
+fn to_egui_color(color: &Color) -> egui::Color32 {
+    let [r, g, b, a] = color.to_array();
+    egui::Color32::from_rgba_unmultiplied(r, g, b, a)
+}
```

Comment:
We already have this:
https://github.com/rerun-io/rerun/blob/7841b5ede6ae613781d7adfe6002287e14332f80/crates/store/re_types/src/components/color_ext.rs#L57-L64